jenkins/Dockerfile-blossom.integration.ubuntu

```diff
@@ -41,7 +42,7 @@ RUN add-apt-repository ppa:deadsnakes/ppa && \
     apt-get update -y && \
     DEBIAN_FRONTEND="noninteractive" apt-get install -y maven \
     openjdk-8-jdk openjdk-11-jdk python3.8 python3.8-distutils python3-setuptools tzdata git wget
-RUN python3.8 -m easy_install pip
+RUN wget https://bootstrap.pypa.io/get-pip.py && python3.8 get-pip.py
```

Comment:
Curious why we're using the default python 3.8 coming from the Ubuntu distribution but manually installing pip outside of the package manager instead of installing it via the python3-pip package?  If this is really necessary, would be good to add a comment explaining why.

Comment applies to the other Dockerfiles where the pattern was copied.

Reply:
The apt-installed python3-pip would install pip for OS default python3 only
like python3.6 in 18.04, python3.10 in 22.04

let me add the comment to make it clear